github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,234 @@
+presubmits:
+  kubevirt/kubevirt:
+  - name: pull-kubevirt-e2e-k8s-1.10.11
```

Comment:
can we remove pull world, it just lacks sense her

Reply:
I took the convention from kubernetes. You will see postsubmit results also on the PR. It helps to differentiate them.